### PR TITLE
Tradermade ws

### PR DESCRIPTION
**Pull request by cds95**
- Add extra endpoint for forex.  `forex` endpoint does the same thing as the `live` endpoint 
 - Enable WS support for `forex` endpoint
 - Modify default config to allow setting `WS_API_KEY` environment variable

### CODE REVIEW

File: packages/sources/tradermade/src/endpoint/forex.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+import * as live from './live'
+
+export const supportedEndpoints = ['forex']
+
+export const execute: ExecuteWithConfig<Config> = async (input, context, config) => {
```
Comment (cds95) [inline packages/sources/tradermade/src/endpoint/forex.ts:6]: This is a bit weird but it looks like we only want to limit WS support for `forex` pairs.

**Review by boxhock — CHANGES_REQUESTED** [review]
High-level this looks good. Could you please also add the WS limits in the `limits.json` file? https://github.com/smartcontractkit/external-adapters-js/blob/develop/packages/core/bootstrap/src/lib/provider-limits/limits.json#L922-L944

File: packages/sources/tradermade/src/endpoint/forex.ts (hunk 1 above)
Comment (boxhock) [reply packages/sources/tradermade/src/endpoint/forex.ts:6]: I'm not sure if I'm missing something, but couldn't we add `forex` in the `supportedEndpoints` array? And shouldn't they be switched around?

Not sure I fully understand when do we want HTTP only vs when do we want to enable WS requests? Isn't it enough with only one endpoint and controlling WS with `WS_ENABLED`?

**Review by boxhock — CHANGES_REQUESTED** [review]

File: packages/sources/tradermade/src/endpoint/forex.ts (hunk 1 above)
Comment (cds95) [reply packages/sources/tradermade/src/endpoint/forex.ts:6]: that's what I had thought a well but it sounded like Bryan wanted equities to keep using http and for only forex to use WS.  I'll confirm with him again during standup.

Comment (cds95) [reply packages/sources/tradermade/src/endpoint/forex.ts]: Confirmed with Bryan that we can collapse this into a single endpoint.

File: packages/sources/tradermade/README.md [hunk 2]
```diff
@@ -1,32 +1,94 @@
 # Chainlink External Adapter for Tradermade
 
+## Endpoints
+
+- `live` and `forex`. Use `forex` when querying for currency pairs. WebSockets can be enabled when fetching data from the `forex` endpoint but not the `live` endpoint. Adapter defaults to the `live` endpoint.
+
```
Comment (boxhock) [inline packages/sources/tradermade/README.md:6]: Can we update this README to follow the format of how we usually document adapters with different endpoints? That would give us some more room to explain each endpoint in their own respective sections

**Review by boxhock — APPROVED** [review]

**Pull request merged**